auth/oidc.go

```diff
@@ -418,7 +418,7 @@ func getJWTUsername(provider JWTConfigCommon, identity *Identity) (username stri
 	if provider.UsernameClaim != "" {
 		value, ok := identity.Claims[provider.UsernameClaim]
 		if !ok {
-			return "", fmt.Errorf("jwt: specified claim %q not found in id_token, identity: %v", provider.UsernameClaim, identity)
+			return "", base.RedactErrorf("jwt: specified claim %q not found in id_token, identity: %+v", base.UD(provider.UsernameClaim), base.UD(identity))
```

Comment:
`provider.UsernameClaim` isn't user data, it's a key name defined in OIDC provider config, rather than the value sent by the client.

```suggestion
			return "", base.RedactErrorf("jwt: specified claim %q not found in id_token, identity: %+v", provider.UsernameClaim, base.UD(identity))
```